lib/python/rose/macro.py

```diff
@@ -246,8 +246,8 @@ def _get_config_sections(self, config_data):
             for key, node in config_data.value.items():
                 if isinstance(node.value, dict):
                     sections.append(key)
-                elif "" not in sections:
```

Comment:
The main point, however, is that without the change, it does not report error for an empty `rose-suite.info`. The change does not break any existing tests otherwise.